sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -4958,10 +4959,34 @@ namespace Azure { namespace Storage { namespace Blobs {
                 x_ms_last_access_time__iterator->second,
                 Azure::Core::DateTime::DateFormat::Rfc1123);
           }
-          auto content_range__iterator = httpResponse.GetHeaders().find("content-range");
-          if (content_range__iterator != httpResponse.GetHeaders().end())
+          auto content_range_iterator = httpResponse.GetHeaders().find("content-range");
+          if (content_range_iterator != httpResponse.GetHeaders().end())
           {
-            response.ContentRange = content_range__iterator->second;
+            const std::string& content_range = content_range_iterator->second;
+            auto bytes_pos = content_range.find("bytes ");
+            auto dash_pos = content_range.find("-", bytes_pos + 6);
```

Comment:
`6` is not pretty.

Reply:
Do we have to be this strict about generated code?

Reply:
No, feel free to leave it as is.